rpm/chruby.spec

```diff
@@ -24,9 +24,8 @@ Changes the current Ruby.
 %build
 
 %install
-make install PREFIX=%{buildroot}/usr
+make install install-zsh-completion PREFIX=%{buildroot}/usr
```

Comment:
Could we make the installation of completion rules optional?

Or, we could check for the existance of `zsh` (`command -v zsh >/dev/null && ...`) and only install zsh completion rules if `zsh` is installed?

Reply:
I’m not sure how that works for binary packages. My understanding is that if I’m installing chruby with

```
dnf install chruby
```

then it’s using a binary package. The package was probably built on some server runner, which doesn’t have Zsh. Do you use Fedora? I can also find some Fedora users to see their packaging policy (like how to make something optional).

Reply:
That's a good point. I believe rpm uses both a build root and an installation root. Ignore checking for the presence of `bash`/`zsh`, but there should also be an `install-bash-completion` target.